host/CMakeLists.txt

```diff
@@ -66,7 +66,14 @@ add_subdirectory("${PROJECT_TOP}/repos/LGMP/lgmp" "${CMAKE_BINARY_DIR}/lgmp"  )
 add_subdirectory(platform)
 
 if(WIN32)
-  add_executable(looking-glass-host WIN32 ${SOURCES})
+  set(CMAKE_RC_FLAGS "-I${PROJECT_TOP}")
```

Comment:
IIRC you can just use a normal `target_include_directories` to do this in a more portable way and not clobber the `CMAKE_RC_FLAGS` var. source properties can work too.

Reply:
For simplicity's sake, I am making this PR just do the `OBJECT_DEPENDS` stuff.